packages/sdk/src/services/presentation-request-service.ts

```diff
@@ -100,6 +100,41 @@ export class PresentationRequestService extends Service<PresentationRequestsApi>
     return itemService.create(itemServiceAuth, newPresentationRequestResponseItem);
   }
 
+  public async getPresentationRequestResponseItem(
+    auth: IVaultToken & IKeystoreToken & IDEK & IKEK,
+    responseId: string
```

Comment:
please change from `responseId ` to `itemId` as it has nothing to do with the response its a vault item ID